azure-stack/user/kubernetes-aks-engine-custom-vnet.md

```diff
@@ -47,6 +47,12 @@ Create a new subnet in your virtual network. You will need to the get the subnet
     
 6. In the subnet blade, make a note of the address range and the virtual network CIDR Block, for example: `10.1.0.0 - 10.1.0.255 (256 addresses)` and `10.1.0.0/24`.
 
+## Considerations for selecting an address space
```

Comment:
Are you sure you are doing the change over the latest doc? I notice that the instruction above your section starts with number 6. in the currently published doc it starts with number 8. See here: https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-custom-vnet?view=azs-2005#create-custom-virtual-network

Reply:
This is on top of the latest master, you can go to Files Changed > Preview. In the preview the numbering is correct. 
You can also see the raw file from the latest master, in the source the numbers shown are 1-6 but in the preview are 1-8. The images take the remaining numbers.

https://raw.githubusercontent.com/MicrosoftDocs/azure-stack-docs/master/azure-stack/user/kubernetes-aks-engine-custom-vnet.md